simulator-starter/src/main/java/com/consol/citrus/simulator/controller/ActivityController.java

```diff
@@ -60,4 +60,10 @@ public void clearExecutions() {
     public ScenarioExecution getScenarioExecution(@PathVariable("id") Long id) {
         return activityService.getScenarioExecutionById(id);
     }
+    
+    @RequestMapping(method = RequestMethod.POST)
+    public Collection<ScenarioExecution> getScenarioExecutions(@RequestBody ScenarioExecutionFilter filter) {
```

Comment:
this replaces the current endpoint, right?

Reply:
Yes. Once the UI is finished we will probably see, that the othe endpoints can be replaced by this generic filtering one.